baybe/utils/plotting.py

```diff
@@ -0,0 +1,78 @@
+"""Plotting utilities."""
+
+import json
+import os
+import sys
+import warnings
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+
+def create_plots(data: pd.DataFrame, path: Path, base_name: str, **kwargs) -> None:
+    """Create plots from a given data frame and save them as a svg file.
+
```

Comment:
```suggestion
def create_example_plots(data: pd.DataFrame, path: Path, base_name: str, **kwargs) -> None:
```

Reply:
Will rename

Reply:
its quite explicitly tied to our repo and examples via the styles used

Reply:
Renamed.